### PR TITLE
📝 PIC-1685: Add common-platform test payloads for review

### PR DESCRIPTION
**Pull request by jhevans**

### CODE REVIEW

File: src/test/resources/messages/common-platform/hearing-with-legal-entity-defendant.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+              "wording": "on 01/08/2009 at  the County public house, unlawfully and maliciously wounded, John Smith"
+            }
+          ],
+          "personDefendant": {
+            "personDetails": {
+              "address": {
+                "address1": "14 Tottenham Court Road",
+                "address2": "London Road",
+                "address3": "England",
+                "address4": "UK"
```
Comment (chris-faulkner-mojdt) [inline src/test/resources/messages/common-platform/hearing-with-legal-entity-defendant.json:48]: There's address5 and postcode

File: src/test/resources/messages/common-platform/hearing-with-legal-entity-defendant.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  "jurisdictionType": "CROWN",
+  "prosecutionCases": [
+    {
+      "defendants": [
+        {
+          "id": "0ab7c3e5-eb4c-4e3f-b9e6-b9e78d3ea199",
+          "pncId": "2004/0012345U",
+          "offences": [
+            {
+              "id": "a63d9020-aa6b-4997-92fd-72a692b036de",
```
Comment (chris-faulkner-mojdt) [inline src/test/resources/messages/common-platform/hearing-with-legal-entity-defendant.json:30]: Not related to this PR but should we start to store this ID ? It seems like we've been told we can rely on these UUIDs so might be useful. We did it for defendant and might be good to do it here.

File: src/test/resources/messages/common-platform/hearing.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    }
+  ],
+  "id": "8bbb4fe3-a899-45c7-bdd4-4ee25ac5a83f",
+  "jurisdictionType": "CROWN",
+  "prosecutionCases": [
+    {
+      "defendants": [
+        {
+          "id": "0ab7c3e5-eb4c-4e3f-b9e6-b9e78d3ea199",
+          "pncId": "2004/0012345U",
```
Comment (chris-faulkner-mojdt) [inline src/test/resources/messages/common-platform/hearing.json:27]: There's also `croNumber`

File: src/test/resources/messages/common-platform/hearing-with-legal-entity-defendant.json (hunk 2 above)
Comment (jhevans) [reply src/test/resources/messages/common-platform/hearing-with-legal-entity-defendant.json:30]: Yeah it's a good question, maybe we should 🤔 I'll raise a placeholder ticket for discussion

**Pull request merged**